onnxruntime/test/onnx/main.cc

```diff
@@ -338,6 +341,14 @@ int real_main(int argc, char* argv[], Ort::Env& env) {
       return -1;
 #endif
     }
+    if (enable_acl) {
+#ifdef USE_ACL
+      Ort::ThrowOnError(OrtSessionOptionsAppendExecutionProvider_ACL(sf,0));
```

Comment:
>0 [](start = 72, length = 1)

hardcode the option here is not good. you can re-use enable_cpu_mem_arena.

Reply:
Fixed in v3